### PR TITLE
Add KNOWN_ISSUES.md document

### PR DESCRIPTION
**Pull request by ivan4th**
The document lists some of the known limitations of UPG and also VPP issues that affect UPG usage

### CODE REVIEW

File: KNOWN_ISSUES.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+PFCP may produce large UDP datagrams sometimes. By
+default, VPP splits datagrams over 1500 into multiple UDP datagrams,
+instead of using proper IP fragmentation. In order to overcome this
+limitation, use "udp { mtu XXX }" option in the VPP command line. The
+value of "UDP MTU" doesn't need to reflect the MTU value of the
+interfaces, it only controls that "UDP splitting" mechanism. If UDP
+MTU is greater than the interface MTU from which the packet is sent,
+the packet will be subject to IP fragmentation.  Up to a certain VPP
+version, there was a bug that was causing corruption of UDP datagrams
+of size greater than 1908 bytes.  The problem is now [fixed][UDPFIX]
```
Comment (jellonek) [inline KNOWN_ISSUES.md:62]: Minor nit - double space.

File: KNOWN_ISSUES.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+in VPP master, and the fix is provided as a downstream patch under
+`vpp-patches/`.
+   
+IP Fragment Size limitation
+---------------------------
+
+VPP can't do buffer chaining for IP fragments it produces.  This means
```
Comment (jellonek) [inline KNOWN_ISSUES.md:69]: Minor nit - double space.

File: KNOWN_ISSUES.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+-------
+
+PFCP may produce large UDP datagrams sometimes. By
+default, VPP splits datagrams over 1500 into multiple UDP datagrams,
+instead of using proper IP fragmentation. In order to overcome this
+limitation, use "udp { mtu XXX }" option in the VPP command line. The
+value of "UDP MTU" doesn't need to reflect the MTU value of the
+interfaces, it only controls that "UDP splitting" mechanism. If UDP
+MTU is greater than the interface MTU from which the packet is sent,
+the packet will be subject to IP fragmentation.  Up to a certain VPP
```
Comment (jellonek) [inline KNOWN_ISSUES.md:60]: Minor nit - double space.

**Review by jellonek — APPROVED** [review]
LGTM

**Review by korroot — APPROVED** [review]
LGTM

**Review by sergeymatov — APPROVED** [review]

**Pull request merged**